### PR TITLE
LE Audio: Media and MCC shell control point improvements

### LINKED ISSUE #50782: LE Audio: The MPL shell module should not use opcodes
**Is your enhancement proposal related to a problem? Please describe.**
When using the `mpl` shell module, it should not use opcode to execute operations, as the user will unlikely know all the opcodes. This makes the `mpl` shell very hard to use. 

**Describe the solution you'd like**
Add `mpl start`, `mpl stop`, `mpl pause`, etc. shell commands to make it more usable by humans. 

**Describe alternatives you've considered**
This maybe should extend into the actual `bt_mcc_send_cmd` API

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Thalley**
Adds explicit functions for the control point operations so that it is easier to use the `mcc` and `media` shell commands. 

fixes https://github.com/zephyrproject-rtos/zephyr/issues/50782

### CODE REVIEW

**Review by kruithofa — DISMISSED** [review]

File: subsys/bluetooth/shell/media_controller.c [hunk 1]
```diff
@@ -720,25 +720,453 @@ static int cmd_media_read_media_state(const struct shell *sh, size_t argc, char
...
+		shell_error(sh, "Invalid track: %ld", track);
+
+		return -ENOEXEC;
+	}
+
+	cmd.param = (int32_t)track;
+
+	err = media_proxy_ctrl_send_command(current_player, &cmd);
+	if (err != 0) {
+		shell_error(sh, "Media Controller move relative failed: %d",
```
Comment (MariuszSkamra) [inline subsys/bluetooth/shell/media_controller.c:1060]:
```suggestion
		shell_error(sh, "Media Controller goto track failed: %d",
```

File: subsys/bluetooth/shell/media_controller.c [hunk 2]
```diff
@@ -720,25 +720,453 @@ static int cmd_media_read_media_state(const struct shell *sh, size_t argc, char
...
+		shell_error(sh, "Invalid group: %ld", group);
+
+		return -ENOEXEC;
+	}
+
+	cmd.param = (int32_t)group;
+
+	err = media_proxy_ctrl_send_command(current_player, &cmd);
+	if (err != 0) {
+		shell_error(sh, "Media Controller move relative failed: %d",
```
Comment (MariuszSkamra) [inline subsys/bluetooth/shell/media_controller.c:1168]:
```suggestion
		shell_error(sh, "Media Controller goto group failed: %d",
```

File: subsys/bluetooth/shell/media_controller.c [hunk 3]
```diff
@@ -720,25 +720,453 @@ static int cmd_media_read_media_state(const struct shell *sh, size_t argc, char
...
+		shell_error(sh, "Invalid segment: %ld", segment);
+
+		return -ENOEXEC;
+	}
+
+	cmd.param = (int32_t)segment;
+
+	err = media_proxy_ctrl_send_command(current_player, &cmd);
+	if (err != 0) {
+		shell_error(sh, "Media Controller move relative failed: %d",
```
Comment (MariuszSkamra) [inline subsys/bluetooth/shell/media_controller.c:950]:
```suggestion
		shell_error(sh, "Media Controller goto segment failed: %d",
```

File: subsys/bluetooth/shell/mcc.c [hunk 4]
```diff
@@ -927,36 +929,439 @@ int cmd_mcc_read_media_state(const struct shell *sh, size_t argc,
...
+		shell_error(sh, "Invalid group: %ld", group);
+
+		return -ENOEXEC;
+	}
+
+	cmd.param = (int32_t)group;
+
+	err = bt_mcc_send_cmd(default_conn, &cmd);
+	if (err != 0) {
+		shell_error(sh, "MCC move relative failed: %d", err);
```
Comment (MariuszSkamra) [inline subsys/bluetooth/shell/mcc.c:1357]:
```suggestion
		shell_error(sh, "MCC goto group failed: %d", err);
```

File: subsys/bluetooth/shell/mcc.c [hunk 5]
```diff
@@ -927,36 +929,439 @@ int cmd_mcc_read_media_state(const struct shell *sh, size_t argc,
...
+		shell_error(sh, "Invalid track: %ld", track);
+
+		return -ENOEXEC;
+	}
+
+	cmd.param = (int32_t)track;
+
+	err = bt_mcc_send_cmd(default_conn, &cmd);
+	if (err != 0) {
+		shell_error(sh, "MCC move relative failed: %d", err);
```
Comment (MariuszSkamra) [inline subsys/bluetooth/shell/mcc.c:1255]:
```suggestion
		shell_error(sh, "MCC goto track failed: %d", err);
```

File: subsys/bluetooth/shell/mcc.c [hunk 6]
```diff
@@ -927,36 +929,439 @@ int cmd_mcc_read_media_state(const struct shell *sh, size_t argc,
...
+		shell_error(sh, "Invalid segment: %ld", segment);
+
+		return -ENOEXEC;
+	}
+
+	cmd.param = (int32_t)segment;
+
+	err = bt_mcc_send_cmd(default_conn, &cmd);
+	if (err != 0) {
+		shell_error(sh, "MCC move relative failed: %d", err);
```
Comment (MariuszSkamra) [inline subsys/bluetooth/shell/mcc.c:1153]:
```suggestion
		shell_error(sh, "MCC goto segment failed: %d", err);
```

**Review by MariuszSkamra — CHANGES_REQUESTED** [review]
Minor issues, otherwise LGTM

**Review by kruithofa — DISMISSED** [review]

**Review by asbjornsabo — COMMENTED** [review]
The comments from @MariuszSkamra must be addressed, otherwise looks good.

**Review by asbjornsabo — APPROVED** [review]

**Review by MariuszSkamra — APPROVED** [review]

**Pull request merged**